src/cache/cache_las.c

```diff
@@ -1270,9 +1281,18 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
             WT_ERR(cursor->next(cursor));
         }
 
-        /* Don't remove the non obsolete entry. */
-        if (las_timestamp > S2C(session)->txn_global.oldest_timestamp ||
-          las_timestamp > S2C(session)->txn_global.last_ckpt_timestamp ||
+        /*
+         * There are several conditions that need to be met
```

Comment:
Please change the comment such that lines wrap at 100 column width wherever you are not starting a new paragraph or writing separate points.

Reply:
Changed.